examples/python/hook_functions.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+#######
+# To allow this module to be imported by other triggers
+# please place this file at /usr/share/foreman/config/
```

Comment:
This is not a general purpose directory, it's part of the Foreman application config - the recommendation should be elsewhere, or preferably in system-wide Python module dir.